packages/account/src/Containers/account.jsx

```diff
@@ -1,14 +1,16 @@
+import 'Styles/account.scss';
+
+import { FadeWrapper, Icon, Loading, PageOverlay, Text, VerticalTab } from '@deriv/components';
+import { PlatformContext, getSelectedRoute, isMobile, matchRoute, routes as shared_routes } from '@deriv/shared';
```

Comment:
This is unnecessary to do unrelated code changes or refactoring in your PRs. It makes it difficult to review.